File: app/src/main/java/telainicial/ActSplash.java
package telainicial;

import android.content.Intent;
import android.os.Bundle;
import android.support.annotation.Nullable;

import base.ActBase;
import custom.ActHost;
import debugging.ExceptionHandler;

public class ActSplash extends ActBase {

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        startActivity(new Intent(this, ActHost.class));
        finish();
    }
}

File: app/src/main/java/telainicial/ActSplash.kt
package telainicial

import android.content.Intent
import android.os.Bundle
import base.ActBase
import base.ActHost

class ActSplash : ActBase() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
//        Thread.setDefaultUncaughtExceptionHandler(ExceptionHandler())
        startActivity(Intent(this, ActHost::class.java))
        finish()
    }
}